addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java

```diff
@@ -207,7 +207,7 @@ public void scan() {
         log.debug(
                 "Checking [{}] [{}], for Relative Path Confusion issues",
                 originalMsg.getRequestHeader().getMethod(),
-                originalMsg.getRequestHeader().getURI().toString());
+                originalMsg.getRequestHeader());
```

Comment:
`.getURI()`

Reply:
In this class there's another log statement that could be tweaked:
```Java
log.debug("The file extension of {} is {}", baseUri.getURI().toString(), fileext);
```
`baseUri.getURI().toString()` → `baseUri`